arches/webpack/webpack-utils/build-vue-filepath-lookup.js

```diff
@@ -0,0 +1,39 @@
+const Path = require('path');
+const fs = require('fs');
+
+const buildVueFilePathLookup = function(path, outerAcc, vueDirectoryPath) {
+    if (!fs.existsSync(path)) {
+        return;
+    }
+    
+    return fs.readdirSync(path).reduce((acc, name) => {
+        const outerPath = vueDirectoryPath || path;   // original `path` arg is persisted through recursion
+        
+        if (fs.lstatSync(Path.join(path, name)).isDirectory() ) {
+            return buildVueFilePathLookup(
+                Path.join(path, name), 
+                acc, 
+                outerPath
+            );
+        }
+        else {
+            let subPath = (Path.join(path, name)).split(/frontend(.*)/s)[1];  // splits only on first occurance
```

Comment:
As you mentioned @chrabyrd, we might want to change this directory as the location for vue components. Maybe something like `../media/vue_components`? In any case we can let the team weigh in and change it to something else before the 7.5 release if we need to.